um_canonicalize/tags.yaml

```diff
@@ -244,6 +278,9 @@ categories:
     - '3'
     - '4'
     - INCL
+    - 1+INCL
+    - 1+EXCL
+    - 3+INCL
```

Comment:
I can't fathom what an inclusive 3rd person would mean. Usually clusitivity distinctions are limited to 1 and/or 2...

Reply:
AFAIK it's also not attested in UniMorph data?

Reply:
It's for Mordvinic languages

Reply:
This is for American Indigenous (Oto-Manguean) languages that we recently added as well.

Reply:
1) Agree on 3+INCL, I will double-check the initial data;
2) Initial tags were like 1PL.INCL but current UniMorph schema suggest to reformat it into PL;1+INCL